test/__init__.py

```diff
@@ -247,6 +249,7 @@ def _init_client(self):
 
                 # May not have this if OperationFailure was raised earlier.
                 self.cmd_line = self.client.admin.command('getCmdLineOpts')
+                self.server_status = self.client.admin.command('serverStatus')
```

Comment:
You only need to run serverStatus once after creating the authenticated client. Just un-indent this line and remove the serverStatus on line 229.